packages/fast-element/src/index.ts

```diff
@@ -13,3 +12,4 @@ export * from "./observation/observable";
 export * from "./behaviors/index";
 export * from "./directives/directive";
 export * from "./directives/bind";
+export * from "./dom";

```

Comment:
After talking with @chrisdholt I realized we weren't exporting the DOM helpers. It's not really related to the main purpose of this PR. I just sort of squeezed this change in since it was so small.